Sender/data_from_csv.cpp

```diff
@@ -0,0 +1,53 @@
+#include"data_from_csv.h"
+
+DataFrom_csv::DataFrom_csv(std::string& fileName): _csvReader(fileName)
+{	if (_csvReader.ReadIsGood)
+	{	_file_lines = _csvReader.GetFileLines();
+		FormatLines_csv(); 
+	}
+}
+void DataFrom_csv::FormatLines_csv()	
+//needs to be changed if input file format is different
+{	UpdateOutputFormat();
+	UpdateMonthAndYear();
+	UpdateFootfallValues();
+}
+void DataFrom_csv::UpdateOutputFormat()
+{	const char* outputFormat = _file_lines[0].c_str();
+	_output_format.UpdateFormat(outputFormat);
+}
+void DataFrom_csv::UpdateMonthAndYear()
+{	std::vector<int> month_and_year = SplitString(_file_lines[1]);
+	_daily_footfalls.UpdateValues(month_and_year[0], month_and_year[1]);
+}
+void DataFrom_csv::UpdateFootfallValues()
+{	std::vector<int> footfall_values = Get_FootfallValues();
+	if (_daily_footfalls.GetNoOfDays() == (int)footfall_values.size())
+		_daily_footfalls.UpdateFootfallPerDay(footfall_values);
+}
+std::vector<int> DataFrom_csv::Get_FootfallValues()
+{	std::vector<int> footfall_values;
```

Comment:
initialize

Reply:
same problem as previous, but even with no file, it runs and gives a default values of 1 for each day.